automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeCreationDiskSpec.scala

```diff
@@ -219,19 +220,29 @@ class RuntimeCreationDiskSpec
           }
         })
         _ <- deleteRuntimeWithWait(googleProject, runtimeName, false)
+        _ <- LeonardoApiClient.patchDisk(googleProject, diskName, UpdateDiskRequest(Map.empty, newDiskSize))
+        _ <- IO.sleep(5 seconds)
```

Comment:
I wonder if there's another way like monitoring/polling something instead of explicit `sleep`ing.

Reply:
`IO.sleep` doesn't block any threads. So I think this is fine....Introducing polling here will add a lot more complexility, which I don't see much benefit in automation tests